parts/22-query.sh

```diff
@@ -9,6 +9,72 @@ _query_long_help="
 	In all cases 'explainquery' will show you the query plan, in case you need to optimise or index data. 'explainjsonquery' is useful with PEV: http://tatiyants.com/pev/
 "
 
+query_longest-running-jobs-by-destination() { ## : get the longest running jobs on the Galaxy server and outputs the longest running jobs by destination in InfluxDB line protocol format
+	handle_help "$@" <<-EOF
+		This script uses gxadmin to get the longest running jobs on the Galaxy server and outputs the longest running jobs by destination in InfluxDB line protocol format.
```

Comment:
```suggestion
		List the longest (currently) running jobs on the Galaxy server
```

- it is gxadmin so don't need to mention that, it isn't another script which calls gxadmin
- this doesn't output influx, it uses gxadmin's query engine which can output influx/tsv/csv/etc which is something that's already known to consumers of this tool.

(this comment applies to both, i only provide suggested wording for one, please make similar changes to the other.)